src/services/claim/MerkleOrchardV1Config.json

```diff
@@ -0,0 +1,73 @@
+{
+  "1": [
+    {
+      "label": "NEXO",
+      "distributor": "0x0000000000000000000000000000000000000000",
+      "token": "0xB62132e35a6c13ee1EE0f84dC5d40bad8d815206",
+      "manifest": "https://raw.githubusercontent.com/balancer/bal-mining-scripts/master/reports/_current-nexo.json",
+      "weekStart": 1
+    },
+    {
+      "label": "BANK",
+      "distributor": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
```

Comment:
```suggestion
      "distributor": "0x9d20FE66eC5Dd15a3D3213556534C77cA20318bE",
```

Reply:
There were 2 BANK distributors, and this is the one that was left in the orchard V1